### DISCUSSION TITLE
i cannot get anything from this pdf，i cannot copy，cannot extract

### DISCUSSION THREAD

**Discussion by No360201** [pymupdf/PyMuPDF#3840] [Looking for help]
### Description of the bug

i cannot get anything from this pdf，i cannot copy，cannot extract
can you help me,thank you
[demo.pdf](https://github.com/user-attachments/files/16860380/demo.pdf)

### How to reproduce the bug

cannot get anything

### PyMuPDF version

1.24.10

### Operating system

Linux

### Python version

3.10

**Comment by JorjMcKie** [comment]
The text cannot be extracted nor copy-pasted because it contains no back-translation info from the visible character images (glyphs) to Unicode.
This applies to all tool doing text extraction.
All you can try is OCRing the file.

**Reply by No360201** [reply]
is there any other solution, after all, the speed and accuracy of ocr are not as good as extraction

**Reply by JorjMcKie** [reply]
No, that's all you can do.
